app/feature/feature-terminate-insurance/src/main/kotlin/com/hedvig/android/feature/terminateinsurance/step/survey/TerminationSurveyDestination.kt

```diff
@@ -183,34 +184,17 @@ private fun TerminationSurveyScreen(
       }
       for (reason in uiState.reasons) {
         Column {
-          HedvigCard(
-            onClick = { selectOption(reason.surveyOption) },
-            colors = CardDefaults.outlinedCardColors(
-              containerColor = MaterialTheme.colorScheme.surface,
-            ),
-            modifier = Modifier
-              .fillMaxWidth()
-              .padding(horizontal = 16.dp),
-          ) {
-            Row(
-              verticalAlignment = Alignment.CenterVertically,
+          com.hedvig.android.design.system.hedvig.HedvigTheme {
+            // todo: where do we apply the theme now that we're still on both old and new theme?
```

Comment:
Same as above, all these can be removed since we got the top-level theme set at the application level.
For previews of course it means we gotta provide the right theme, especially since in the DS module itself we only get access to one of the two themes and it makes the live template a bit trickier to use for now